browser-interface/packages/shared/comms/sagas.ts

```diff
@@ -410,7 +410,9 @@ async function stripSnapshots(profile: Avatar): Promise<Avatar> {
     ...profile,
     avatar: { ...profile.avatar, snapshots: newSnapshots as Snapshots },
     // THIS IS IMPORTANT, the blocked and muted sizes are too big for the network and are unnecesary
-    blocked: [],
+    // blocked status propagation has been re-enabled so the current user knows when a peer has blocked it
```

Comment:
can't messages be ignored locally instead? I'm worried about the big warning above

Reply:
I meant the warning in the comment "the blocked and muted sizes are too big for the network", can we simply ignore the messages if we get one from a blocked user?

Reply:
Is not only the messages. You shouldnt be able to interact with friend requests or preview the passport of peers that blocked you. We have special states in the UI for this.

Reply:
ok, understood, but I don't think we can simply send this through comms, some users have thousands of addresses in the block list, and this message is broadcasted to everyone around the user.

Reply:
I will try to make a workaround by fetching the user profile from the catalyst when this information is required

Reply:
Trying to fetch the blocked list from the catalyst when needed adds too much complexity thus a mess to handle with the current architecture. A new proposal is coming on which profile information will always be fetched from the catalyst instead of being broadcasted through comms. This way would fix the problem